pkg/plugins/trivy/plugin.go

```diff
@@ -661,17 +664,10 @@ func (p *plugin) getPodSpecForStandaloneMode(ctx trivyoperator.PluginContext, co
 			TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
 			Env:                      env,
 			Command: []string{
-				"trivy",
+				"/bin/sh",
 			},
 			Args: []string{
-				"--cache-dir",
-				"/tmp/trivy/.cache",
-				"--quiet",
-				"image",
-				"--skip-update",
-				"--format",
-				"json",
-				optionalMirroredImage,
+				"-c", fmt.Sprintf("trivy image %s --cache-dir /tmp/trivy/.cache --quiet  --skip-update --format json > /tmp/scan/result.json &&  bzip2 -c /tmp/scan/result.json | base64", optionalMirroredImage),
```

Comment:
Has the value of `optionalMirroredImage` been validated beforehand? Just making sure someone doesn't do anything funky with that now that we're using it for string interpolation.